Duplicati/Library/Backend/S3/S3IAM.cs

```diff
@@ -162,12 +158,12 @@ private string GeneratePolicyDoc(string path)
 
                 dict = new Dictionary<string, string>
                 {
-                    ["isroot"] = "False", //user.Arn.EndsWith(":root", StringComparison.Ordinal).ToString();
+                    ["isroot"] = "False",

```

Comment:
Only a question:
is this necessary when in line 166 the value are overwritten?

`["isroot"] = (cl.SimulatePrincipalPolicy(new SimulatePrincipalPolicyRequest { PolicySourceArn = user.Arn, ActionNames = new[] { "iam:CreateUser" }.ToList() }).EvaluationResults.First().EvalDecision == PolicyEvaluationDecisionType.Allowed).ToString()`

Reply:
I looked over this, but of course not. Apparently in two places exceptions can occur, I will make more explicit according to what happened in the code before this PR.